app/Models/Mship/Concerns/HasCTSAccount.php

```diff
@@ -26,8 +44,33 @@ public function syncToCTS()
 
             $ctsAccount = DB::table("{$ctsDatabase}.members")->where('cid', $this->id)->first();
 
+            // if no CTS account exists, lets sync to create a new one, and not update anything this time round.
             if (! $ctsAccount) {
-                // No user exists. Abort.
+                // for a division member, use the join timestamp of them joining, else use an empty timestamp.
+                // this is how CTS in its wisdom puts this column on visiting controllers.
+                $is_visitor = $this->primary_permanent_state->code != 'DIVISION';
+                $joined_div = ! $is_visitor
+                    ? $this->primary_permanent_state->pivot->start_at
+                    : gmdate('Y-m-d H:i:s');
+
+                $newMember = [
+                    'old_rts_id' => 0,
+                    'id' => $this->generateCTSInternalID($this->id),
+                    'cid' => $this->id,
+                    'name' => $this->real_name,
```

Comment:
Does this honour nicknames?

Reply:
Probably not, but existing syncing via cron does not from what I've seen (happy to be wrong there)

Reply:
Let's honour nicknames as we do for other services like forum!

Reply:
Done! https://github.com/VATSIM-UK/core/pull/2231/commits/6499524184d03b87e6ba300ad45f727fc9c87018